Sources/Overload/OvEditor/src/OvEditor/Main.cpp

```diff
@@ -74,10 +77,30 @@ int main(int argc, char** argv)
 	}
 
 	if (ready)
+		TryRun(projectPath, projectName);
+
+	return EXIT_SUCCESS;
+}
+
+void TryRun(std::string projectPath, std::string projectName)
```

Comment:
I would suggest the use of `const string& projectPath, const string& projectName` as parameters. Other than that, looks goot to me 👍

Reply:
Good eye, the update is pushed.
I've also made TryRun `static` so it cannot escape from main.cpp.